chalicelib/checks/wrangler_checks.py

```diff
@@ -1177,9 +1192,7 @@ def users_with_doppelganger(connection, **kwargs):
             cases.append(log)
         # if not, compare names
         else:
-            # matcher = stringmatch.Levenshtein()
-            # score = round(matcher.get_sim_score(us1['display_title'], us2['display_title']) * 100)
-            score = fuzz.token_sort_ratio(us1['display_title'], us2['display_title'])
+            score = round(string_label_similarity(us1['display_title'], us2['display_title']) * 100)
```

Comment:
```
>>> s1="""'Arthur Dean' <arthur.dean@bar.foo.com>"""
>>> s2="""'Dean Arthur' <dean.arthur@foo.bar.com>"""
>>> t1=[t.lower() for t in re.split(r'\W+', s1) if t]
>>> t2=[t.lower() for t in re.split(r'\W+', s2) if t]
>>> t1
['arthur', 'dean', 'arthur', 'dean', 'bar', 'foo', 'com']
>>> t2
['dean', 'arthur', 'dean', 'arthur', 'foo', 'bar', 'com']
>>> t1.sort()
>>> t2.sort()
>>> t1
['arthur', 'arthur', 'bar', 'com', 'dean', 'dean', 'foo']
>>> t2
['arthur', 'arthur', 'bar', 'com', 'dean', 'dean', 'foo']
```

Also:

```
>>> s1="""How to turn lead into gold"""
>>> s2="""How to turn gold into lead"""
>>> s3="""To turn into lead gold: how?"""
>>> t1=[t.lower() for t in re.split(r'\W+', s1) if t]
>>> t2=[t.lower() for t in re.split(r'\W+', s2) if t]
>>> t3=[t.lower() for t in re.split(r'\W+', s3) if t]
>>> t1
['how', 'to', 'turn', 'lead', 'into', 'gold']
>>> t2
['how', 'to', 'turn', 'gold', 'into', 'lead']
>>> t3
['to', 'turn', 'into', 'lead', 'gold', 'how']
>>> t1.sort()
>>> t2.sort()
>>> t3.sort()
>>> t1
['gold', 'how', 'into', 'lead', 'to', 'turn']
>>> t2
['gold', 'how', 'into', 'lead', 'to', 'turn']
>>> t3
['gold', 'how', 'into', 'lead', 'to', 'turn']
```
Are those really things you want to count as dopplegangers? It might be that you're just preserving some prior status quo where sorting was done, but I'm not sure. If not, I would just ask you to make sure you're accomplishing what you think you are meaning to accomplish here. I don't know the requirements so it's hard for me to evaluate. At some level, all code just does what it's programmed to do. :)